catalog/transformer/catalog-transformer-service-atom/src/main/java/ddf/catalog/transformer/response/query/atom/AtomTransformer.java

```diff
@@ -146,19 +149,8 @@ public BinaryContent transform(SourceResponse sourceResponse, Map<String, Serial
           "Cannot transform null " + SourceResponse.class.getName());
     }
 
-    Date currentDate = new Date();
-
-    ClassLoader tccl = Thread.currentThread().getContextClassLoader();
```

Comment:
❓ Why was this behavior removed? It is inconsistent with tccl usage elsewhere in this class.

Reply:
Not removed, just moved into the createFeed function